src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -357,6 +370,10 @@ private void HandleRegularCommand(StartPageListItem item)
                     dvm.ShowNewFunctionDialogCommand.Execute(null);
                     break;
 
+                /*case ButtonNames.ShowGallery:
+                    dvm.ShowGalleryCommand.Execute(null);
+                    break;*/
+
```

Comment:
Sorry for not catching this earlier, can we just do nothing internally inside `ShowGalleryCommand`? This way the above code can be enabled and yet we won't have it showing up (also the `dynamoViewModel.ShowGalleryCommand.Execute` right below). Just use a single place to disable the gallery.